Assignment-1.c

```diff
@@ -0,0 +1,135 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <ctype.h>
+
+
+int Isdigit(char ch) {

```

Comment:
Use consistent naming conventions either camelCase or snake_case. like Isdigit to isDigit or perform_operation to performOperation